src/amo/components/CollectionManager/index.js

```diff
@@ -179,6 +180,7 @@ export class CollectionManagerBase extends React.Component<Props, State> {
       collectionSlug: collection.slug,
       editing: true,
       errorHandlerId: errorHandler.id,
+      page: page || 1,
```

Comment:
Do we have a test case for the `|| 1`?

Reply:
We don't have any test cases for this part of the code right now, which is why I opened issue #4982. I'm not sure how that happened, but I hope to address it soon via that issue.